keystone.js

```diff
@@ -11,3 +12,16 @@ keystone.start({
   onMount () { console.log('Application Mounted'); },
   onStart () { console.log('Application Started'); },
 });
+
+var express = require('express');
+var app = express();
+var server = require('http').createServer(app);
+buildSocketIO(server, {
+  allowedOrigins: process.env.NODE_ENV === 'development' ? '*' : [
+    'https://staging-cms.twreporter.org',
+    'https://cms.twreporter.org',
```

Comment:
Do we need to include the CMS domain for porting?

Reply:
I'll remove these cms origins